tools/v8-prof/windows-tick-processor.bat

```diff
@@ -0,0 +1,42 @@
+@echo off
+
```

Comment:
Usually a good idea to put a `setlocal` at the beginning of the script, unless you want the variables you set in the script to be available in the environment afterwards